src/js/themes/prism.js

```diff
@@ -23,7 +23,7 @@ const colors = {
   },
   char: { dark: hpeColors['teal!'], light: hpeColors.teal.dark },
   code: hpeColors.text,
-  comment: { dark: hpeColors.green.light, light: hpeColors.green.dark },
+  comment: { dark: hpeColors.green.light, light: hpeColors['text-primary'] },
```

Comment:
@jcfilben For now, can we just hard-code the HEX value here?

Once we do v1 tokens release, we'll introduce the `text-primary` color with light and dark support and can come back here to update. But since the namespace is still a little in flux, let's avoid introducing it early.

Reply:
Fixed